### ISSUE TITLE
csPlugin: tekstikentän merkkilaskurin varoitusteksti ei lokalisoidu oikein tai sitä ei voi asettaa

### ISSUE THREAD

**Issue by saviit** [TIM-JYU/TIM#3413]
csPlugin-tekstikentän merkki/sanalaskurin (timApp/modules/cs/js/editor/countboard.ts) punainen varoitusteksti ei lokalisoidu tai sitä ei pysty asettamaan markupilla, jos lasketaan merkkimääriä (`tooManyText`, `tooFewText`). Attribuutit `tooManyText`, `tooFewText` eivät itse asiassa tällä hetkellä näytä vaikuttavan mihinkään: jos ei löydy sanamäärän vastaavia niin hypätään oletusvaroitukseen (jonka lokalisointi on rikki).

Sanamäärää laskiessa (`tooManyWord`, `tooFewWord`) toimii, joten sen avulla tuota voi kiertää. Selkeämpää kuitenkin olisi, jos olisi joko

1) vain yksi yleinen attribuutti tuota varten (`tooMany`, `tooFew`)
tai
2) kullekin selkeästi oma attribuutti (`tooManyChars`, `tooManyWords`, jne.), jotka vaikuttavat vain oman tyyppiseensä varoitukseen.

**Comment by vesal** [comment]
Periaattessahan csPluginissa on attribuutti lang:
jolla on olemassa toistaiseksi vain fi ja en toteutus.
Jos tekee uusia attribuutteja, niin voisi yrittää
lisätä niitä (ja tarkistaa vanhoja) niin ettö
jos attribuutille ei anneta arvoa, se tottelee oeltusta lang: - attribuutin
mukaisesti

**Comment by saviit** [comment]
Joo, unohdinkin että oli tuo lang-attribuutti. Tosin nopealla testauksella plugin tottelee ainoastaan käyttäjän TIM-kieliasetusta tuossa, lang-attribuutilla ei näyttänyt olevan mitään vaikutusta. Eli lokalisointi on ehkä kuitenkin tuossa oikein?

Mutta tuo on varmaan kyllä ihan joku logiikkabugi, että ohitetaan noita tekstin pituuteen/merkkien määrään viittaavia attribuutteja.

**Comment by vesal** [comment]
En tiedä lang-attribuutin nykykohtaloa, koska csPlugin on kokenut kovia
refaktorointeja sen jälkeen kun tuon tein.  Eli sen voisi jossakin
yhteydessä tarkistaa.  Idea kuitenkin oli että minimissään
käännökseksi riittäisi laittaa esim lang: en
Toteutus oli paljon ennen i18n aikaa joten se ei noudata
tuota ja sen tehokkuuttaa ei ole kunnolla tutkittu.  Nythän
esim Ohj1 monisteessa on valtava määrä noit csPlugineja ja jos
jokaisessa on paljon attribuutteja joita joka kerta iffitellään
kielen mukaan, niin voi olla hidasta.
